contracts/stellar-axelar-operators/src/migrate.rs

```diff
@@ -0,0 +1,25 @@
+use soroban_sdk::{Address, Env, Vec};
+
+mod storage {
+    use soroban_sdk::{contracttype, Address};
```

Comment:
why is it necessary to import contracttype here? Is it missing from the code generation?